jobs/integr8ly/msb-integration-test.yaml

```diff
@@ -0,0 +1,181 @@
+---
+- job:
+    name: msb-integration-test
+    display-name: 'Managed Service Broker Integration Test'
+    project-type: pipeline
+    concurrent: false
+    triggers:
+      - timed: "15 01 * * *"
+    properties:
+      - build-discarder:
+          num-to-keep: 15
+    parameters:
+      - string:
+          name: 'installationGitUrl'
+          default: 'https://github.com/integr8ly/installation.git'
+          description: '[REQUIRED] Integreatly installer Git URL'
+      - string:
+          name: 'installationGitBranch'
+          default: 'master'
+          description: '[REQUIRED] The name of the git branch to be used for installing Integreatly'
+    dsl: |
+        def testOptions = [:]
+        final QE_REPO_URL = 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
```

Comment:
Is this `QE_REPO_URL` used anywhere?

To be honest I am not sure if these lines of code around `testOptions` are indeed required here. It is part of the Cluster Create and Integreatly Install jobs anyway.  But I am not familiar with the stuff enough so it might be good having it here.

Reply:
It's not. It can be removed.